src/network/networkd-link.c

```diff
@@ -942,6 +942,21 @@ static int link_push_ntp_to_dhcp_server(Link *link, sd_dhcp_server *s) {
         return sd_dhcp_server_set_ntp(s, addresses, n_addresses);
 }
 
+static int link_set_bridge_fdb(Link *link) {
+        FdbEntry *fdb_entry;
+        int r = 0;
+
+        LIST_FOREACH(static_fdb_entries, fdb_entry, link->network->static_fdb_entries) {
+                r = fdb_entry_configure(link, fdb_entry);
+                if (r < 0) {
+                        log_link_error_errno(link, r, "Failed to add MAC entry to static MAC table: %m");
```

Comment:
`return log_link_error_errno(...);`

(Even though you're just moving the code, that's a nice cleanup.)